src/test/junitParser.test.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from 'node:test';
+import { parseTestReports } from '../junitParser';
+
+const DIRNAME = __dirname;
+
+test('should parse junit report with 3 suites', async (t: any) => {
+    const testReports = await parseTestReports('test', 'test', `${DIRNAME}/xml/testsuites.xml`, []);
+    t.assert.snapshot(testReports.length);
```

Comment:
Why is this a snapshot assertion?
```suggestion
    t.assert.strictEqual(testReports.length, 3);
```